styleguide/source/_patterns/01-molecules/01-layouts/08-layout-four-up/layout-four-up.twig

```diff
@@ -0,0 +1,10 @@
+<div class="layout layout-example layout-four-up">
+  <div class="layout-four-up_row">
+    <div class="layout_item layout_container layout-four-up_primary">1</div>
```

Comment:
It seems complicated for each item to have its own class. My goal for this markup is for each item to be as consistent as possible so that we have to push less logic into our CMS implementation.